docs/admin-guide/configuration.md

```diff
@@ -12,8 +12,8 @@ The following environment variables are available to configure the application:
 | [`REDIS_HOST`](#redis_host)                                               | The Redis host.                                                          | No       | `localhost`        |
 | [`REDIS_PORT`](#redis_port)                                               | The Redis port.                                                          | No       | `6379`             |
 | [`REDIS_PASSWORD`](#redis_password)                                       | The Redis password.                                                      | No       | -                  |
-| [`SENTENCE TRANSFORMERS_HOME`](#sentence_transformers_home)               | The directory containing Sentence Transformers data files.               | No       | `hf_data`          |
-| [`SENTENCE_TRANSFORMERS_CONCURRENCY`](#sentence_transformers_concurrency) | The maximum number of concurrent downloads when installing transformers. | No       | `3`                |
+| [`HF_HOME`](#hf_home)                                                     | The directory containing Huggingface Transformers data files.            | No       | `hf_data`          |
+| [`HF_DOWNLOAD_CONCURRENCY`](#hf_download_concurrency)                     | The maximum number of concurrent downloads when installing transformers. | No       | `3`                |
```

Comment:
These are no longer in alphabetical order. Could you please move them to the correct place?